test/example-app/index.js

```diff
@@ -46,20 +46,16 @@ const legacyScriptingSource = `
       },
 
       contact_full_poll: function(bundle) {
-        var response = z.request({
```

Comment:
In this old pattern (which is typical), would we automatically add auth data in WB? I honestly think not, but just making sure.

Reply:
You're right. We don't add auth data automatically to `z.request` in WB.

Reply:
Would that block this PR?

Reply:
No, it's fine then! If we did, we'd have to make sure a `z.request` without using the `bundle.request` (which was "massaged") would work.